test/runtests.jl

```diff
@@ -26,7 +26,7 @@ max_worker_rss != typemax(Csize_t) && move_to_node1("parallel")
 cd(dirname(@__FILE__)) do
     n = 1
     if net_on
-        n = min(8, Sys.CPU_CORES, length(tests))
```

Comment:
Will that set Sys.CPU_CORES and hence also change the number of tests run simultaneously? If so, we should just make this `min(Sys.CPU_CORES, length(tests))`.